statusHandler/view/termuic/termuiConsole.go

```diff
@@ -95,14 +101,43 @@ func (tc *TermuiConsole) eventLoop() {
 func (tc *TermuiConsole) processUiEvents(e ui.Event) {
 	switch e.ID {
 	case "<Resize>":
-		payload := e.Payload.(ui.Resize)
-		tc.grid.SetRectangle(0, 0, payload.Width, payload.Height)
-		ui.Clear()
-		ui.Render(tc.grid.TopLeft(), tc.grid.TopRight(), tc.grid.Bottom())
-
+		tc.doResizeEvent(e)
 	case "<C-c>":
 		ui.Close()
 		StopApplication()
 		return
 	}
 }
+
+func (tc *TermuiConsole) doChanges(counter *uint32) {
+	atomic.AddUint32(counter, 1)
+	if atomic.LoadUint32(counter) > numOfTicksBeforeRedrawing {
+		tc.doFakeResize()
```

Comment:
Maybe you dont need method doFakeResize. I think its ok just call method doResize, and maybe you can change name of method doChange in doFakeResizeIfNeeded.

Reply:
remove that method. Related to the name of `doChanges` I would leave it like this for now because if I would name it `doFakeResizeIfNeeded` the name of the method would shadow the event that happens more often, the refresh